test/libraries/TickMath.t.sol

```diff
@@ -15,8 +15,6 @@ contract TickMathTestTest is Test, JavascriptFfi, GasSnapshot {
     uint160 constant MIN_SQRT_PRICE = 4295128739;
     uint160 constant MAX_SQRT_PRICE = 1461446703485210103287273052203988822378723970342;
```

Comment:
i feel like these constants should also just be used directly from TickMath.sol?
`MIN_SQRT_PRICE = TickMath.MIN_SQRT_PRICE`
etc